File: core-algorithms/2_search/1_linear_search/linear_search.ts
const linearSearch = (arr: number[], targetElement: number) => {
  let pos = -1;
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === targetElement) {
      pos = i + 1;
      break;
    }
  }
  return pos;
};

const linearPos = linearSearch(
  [1, 2, 55, 789, 432, 244, 7899, 8002, 1001],
  8002,
);

console.log(linearPos);

File: core-algorithms/2_search/1_linear_search/linear_search.py
def linearPos(lis, target):
    pos = -1
    for i, value in enumerate(lis):
        if(value == target):
            pos = i
    return pos

linearPos = linearPos(
  [1, 2, 55, 789, 432, 244, 7899, 8002, 1001],
  8002,
);

print(linearPos);